### ISSUE TITLE
Correct the design of the language view component

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#76]
### Brief bug description

The component returns inapropriate data, e.g. instead of `/en-us/doctors/` and `/es-es/medicos/` it returns `/en-us/doctors/` and '/es-es/doctors/`.

### Repro steps

2. Click on '...'
3. Scroll down to '...'

### Expected behavior

What is the correct behavior?

### Test environment

 - Platform/OS: [e.g. .NET Core 3.1, iOS, Windows]
 - Browser [e.g. Chrome, Safari, Edge Chromium, Edge]
 - Version [e.g. 81]

### Additional context

Add any other context about the problem here. If you found an issue where a similar bug has been reported, add a link here.

### Screenshots

Add links to screenshots, if you deem it appropriate.

### Issue expands upon

This issue expands upon another issue(s): #`issue number`

### Blockers

This issue is blocked by the following issue(s): #`issue number`
